components/CommonButton/index.js

```diff
@@ -2,32 +2,55 @@ import styled from "styled-components";
 
 export default function CommonButton({
```

Comment:
If it's a usual button for your app it's ok to name it just `Button`

Reply:
Done  :)